src/plugins/condition/utils/styleUtils.js

```diff
@@ -64,7 +64,7 @@ const styleProps = {
     }
 };
 
-const aggregateStyleValues = (accumulator, currentStyle) => {
+function aggregateStyleValues(accumulator, currentStyle) {
```

Comment:
be careful when replacing arrow style function definitions with the standard function definition, arrow style functions are useful to enclose the context of where it was written.

Reply:
My understanding is that they are interchangeable if the function doesn't use `this` or `arguments` or are called with `new` keyword. This one seems to pass the test. Let me know if I'm missing something.

Reply:
@deeptailor is just giving a word of caution. In this case yes, changing to function declaration is fine.